docs/workforce-management-framework/topics/con-wfm-intergrations.adoc

```diff
@@ -1,26 +1,26 @@
-= {Feature} intergration
+//= {PROJECT_NAME} intergration
 
-An intergration is the use of existing opensource projects to solve specific needs for {Feature}. 
-{Feature} has a number of intergrations that leverage other opensource projects for there expertises in paticular fields.
-This enabled the inclusion of domain expertises in the areas of security and sync in the developement of {Feature}.
+An intergration is the use of existing opensource projects to solve specific needs for {PROJECT_NAME}. 
+{PROJECT_NAME} has a number of intergrations that leverage other opensource projects for there expertises in paticular fields.
+This enabled the inclusion of domain expertises in the areas of security and sync in the developement of {PROJECT_NAME}.
 
-== Security intergrations
+//== Security intergrations
```

Comment:
Why we commenting this? Just want to understand? 
Do you want to hide this for the moment? If yes it seems reasonable.

Reply:
The same header in master.adoc so seeing it twice

Reply:
Why not remove this completely then. 
I'm just seeing most of this for the first time and I think we need to minimize confusion by reducing number of files and references. Docs are small but they overloaded by references.

Reply:
agree will remove the content as it adds nothing , and just clutters up the toc when rendered.